pkgs/development/interpreters/luajit/default.nix

```diff
@@ -94,16 +103,16 @@ stdenv.mkDerivation rec {
       lua = self;
       inherit (luaPackages) requiredLuaModules;
     };
-    withPackages = import ../lua-5/with-packages.nix { inherit buildEnv luaPackages;};
+    withPackages = import ../lua-5/with-packages.nix { inherit buildEnv luaPackages; };
     pkgs = luaPackages;
     interpreter = "${self}/bin/lua";
   };
 
   meta = with lib; {
     description = "High-performance JIT compiler for Lua 5.1";
-    homepage    = "http://luajit.org";
-    license     = licenses.mit;
-    platforms   = platforms.linux ++ platforms.darwin;
+    homepage = "http://luajit.org";
+    license = licenses.mit;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
This could/should be unix.

Reply:
Even `all` I suppose... at least the support upstream is very wide: https://luajit.org/luajit.html
But for Hydra this won't do any change, as we only have linux and darwin there (currently).